Companion to Pypeit PR#2013

tbowers7 opened the pull request:
As titled.  Adds a Coadd2D test for Soar/Goodman that would have failed prior to https://github.com/pypeit/PypeIt/pull/1976.

Companion to https://github.com/pypeit/PypeIt/pull/2013

tbowers7 committed:
Add a Coadd2D test for Soar/Goodman

This is a test for the ``allowed_extensions`` fix in https://github.com/pypeit/PypeIt/pull/2013

	new file:   coadd2d_files/soar_goodman_blue_m1.coadd2d
	modified:   test_scripts/test_setups.py

rcooke-ast commented on test_scripts/test_setups.py:
```diff
@@ -240,9 +239,14 @@ class TestPhase(Enum):
     }
 
+# TODO: Test the pypeit_coadd_datacube setups!
+_coadd3d = {}
```
This is currently done elsewhere... good call to move it here 👍

rcooke-ast reviewed (approved):
Looks good to me, and happy to approve. Please merge once the dev suite passes

tbowers7 committed:
Fix Soar/Goodman coadd2d file

Remove explicit directory structure parts of the Coadd2D file to align
with other files in the DevSuite.

	modified:   coadd2d_files/soar_goodman_blue_m1.coadd2d
